product_docs/docs/biganimal/release/using_cluster/05c_upgrading_log_rep.mdx

```diff
@@ -0,0 +1,225 @@
+
+### Setting up Publication 
+
+To set up the publication, log into the old BigAnimal instance and create a publication:
+
+```sql
+CREATE PUBLICATION <pub_name>;
+```
+
+In this example:
+
+
+```sql
+CREATE PUBLICATION v12_pub;
+```
+
+The expected output is `CREATE PUBLICATION`.
+
+Next, configure the publication to include the specific tables you want replicated on the new instance:
+
+```sql
+ALTER PUBLICATION <pub_name> ADD TABLE <table_name>;
+```
+
+Which in the current example is:
+
+```sql
+ALTER PUBLICATION v12_pub ADD TABLE pgbench_accounts;
+ALTER PUBLICATION v12_pub ADD TABLE pgbench_branches;
+ALTER PUBLICATION v12_pub ADD TABLE pgbench_history;
+ALTER PUBLICATION v12_pub ADD TABLE pgbench_tellers;
+```
+
+Each of these alterations produces `ALTER PUBLICATION` output upon success. 
+
+### Creating the Logical Replication Slot
+
+Then, on the older versioned instance, create a replication slot using the 'pgoutput' plugin:
+
+```sql
+SELECT pg_create_logical_replication_slot('<slot_name>','pgoutput');
+```
+
+In the current example:
+
+```sql
+SELECT pg_create_logical_replication_slot('v12_pub','pgoutput');
+```
+
+If successful, the expected output is something like the following:
+
+```
+ pg_create_logical_replication_slot 
+------------------------------------
+ (v12_pub,0/AC003330)
+```
+
+This slot tracks changes to the published tables on the old instance to ensure they can be replicated to the subscriber on the new instance without losing any data.
+
+
+### Setting up the Subscription
+
+Now, logged into the newer Postgres instance, create a subscription to the publication on the older instance in the following format:
+
+```sql
+CREATE SUBSCRIPTION <name_of_subscription> CONNECTION 'user=<old_instance_username> host=<old_instance_read/write_host> sslmode=require port=<old_instance_port> dbname=<old_instance_dbname> password=<old_instance_password>' PUBLICATION <publication_name> WITH (enabled=true, copy_data = true, create_slot = false, slot_name=<slot_name>);
+```
+
+Specifically, in this example:
+
+```sql
+CREATE SUBSCRIPTION v16_sub CONNECTION 'user=edb_admin host=p-x67kjhacc4.pg.biganimal.io sslmode=require port=5432 dbname=edb_admin password=XXX' PUBLICATION v12_pub WITH (enabled=true, copy_data = true, create_slot = false, slot_name=v12_pub);
+```
+
+Look for the expected output: `CREATE SUBSCRIPTION`.
+
+In this example, the subscription uses a connection string to specify the source database and includes options to copy existing data and to follow the publication identified by 'v12_pub'. 
+
+The subscription mechanism pulls schema changes (with some exceptions, as noted in the PostgreSQL documentation on Limitations of Logical Replication: https://www.postgresql.org/docs/current/logical-replication-restrictions.html) and data from the source to the target database, effectively replicating the data.
```

Comment:
to:  The subscriber will pull data changes from the publisher.  Schema changes can be pulled as well, with exceptions to some DDL commands.  For more information on limitations with schema changes, see:  (link)

Reply:
👍